model.mmd

```diff
@@ -0,0 +1,34 @@
+sequenceDiagram
+	User->>+Browser: enters name and clicks `Let's get started!`
+  Browser->>+Controller: POST /name
```

Comment:
Love this MVC diagram, very clean and clear to look at! 

Only thing I thought was perhaps needs the type of POST data included? i.e. POST /name formdata